routers/private/internal.go

```diff
@@ -50,6 +51,7 @@ func bind(obj interface{}) http.HandlerFunc {
 // These APIs will be invoked by internal commands for example `gitea serv` and etc.
 func Routes() *web.Route {
 	r := web.NewRoute()
+	r.Use(chi_middleware.RealIP)
```

Comment:
It should be put after all other checks (in case some checks depends on the remote addr in the future)

And it needs enough logs: what's the purpose of using it, why it's safe to use it.

Reply:
Done.